app/views/courses/show.html.erb

```diff
@@ -101,9 +101,9 @@
         <%= render partial: 'courses/description_content' %>
       </div>
       <% unless course.not_running? %>
-        <div class="govuk-grid-column-one-third" data-qa="course__status_panel">
+        <aside class="govuk-grid-column-one-third" data-qa="course__status_panel">
```

Comment:
I don't think we should be switching from `div` to `aside`.

Aside:
> a portion of a document whose content is only indirectly related to the document's main content

I would not say that the content of these side columns is indirectly related to the document. In the case of the status panel it's crucial.